notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb

```diff
@@ -1000,7 +952,7 @@
         "\n",
         "You can get the signatures of your model's input and output layers by reloading the model into memory, and querying it for the signatures corresponding to each layer.\n",
         "\n",
-        "When making a prediction request, you need to route the request to the serving function instead of the model, so you need to know the input layer name of the serving function -- which you will use later when you make a prediction request.\n",
+        "When making a prediction request, you need to route the request to the serving function instead of the model, so you need to know the input layer name of the serving function -- which you use later when you make a prediction request.\n",
```

Comment:
Change "so" to "therefore"

- "...therefore, you need to know the input layer name..."